database/query/repositories.sql

```diff
@@ -30,7 +30,7 @@ SELECT * FROM repositories
 WHERE provider = $1 AND project_id = $2
   AND (repo_id >= sqlc.narg('repo_id') OR sqlc.narg('repo_id') IS NULL)
 ORDER BY project_id, provider, repo_id
-LIMIT sqlc.narg('limit');
+LIMIT sqlc.narg('limit')::bigint;
```

Comment:
@dmjb can you double check this change please?

Reply:
> do we ever want to do a single select that fetches all repositories

Technically, we shouldn't, but the minder CLI uses this endpoint, and the CLI still needs to support paginating results.

> do we want to force clients to paginate?

We don't force the client to paginate.

Reply:
> > do we want to force clients to paginate?
>
> We don't force the client to paginate.

My point was that we _could_ enforce this in the API.

But, we'll do that in a subsequent PR.